index.html

```diff
@@ -21,6 +21,15 @@ <h1>Henning Schwichow</h1>
     <figcaption>Random image</figcaption>
   </figure>
   
+  <dl>
```

Comment:
Hey Henning, to be honest I´m not sure :) Normally a `dl` tag is used for definitions. The common lists are with an `ul` or `ol` tag. So I would recommend a `ul` list. The bullet points can be removed in css later ;) What you think?

Reply:
it´s quite old but maybe helps a little bit:dl vs ul

Reply:
Here´s the link [dl vs ul](https://stackoverflow.com/questions/21404774/what-is-the-difference-between-dl-dd-dt-and-ul-li/21404872) :)

Reply:
thanks :)